test/_generate_expected_node.js

```diff
@@ -1,30 +1,85 @@
-const canvg = require("../src/canvg.js"),
-  Canvas = require("canvas"),
-  path = require("path"),
-  Promise = require("bluebird"),
-  fs = Promise.promisifyAll(require("fs")),
-  PNGImage = Promise.promisifyAll(require("pngjs-image"));
+const canvg = require('../src/canvg.js'),
+  Canvas = require('canvas'),
+  path = require('path'),
+  Promise = require('bluebird'),
+  fs = Promise.promisifyAll(require('fs')),
+  PNGImage = Promise.promisifyAll(require('pngjs-image'));
 
-const file = process.argv[2];
-const fileName = path.resolve(`${__dirname}/../svgs/${file}`);
-if (!fs.existsSync(fileName)) {
-  throw new Error(`${fileName} does not exist!`);
+function createCanvas (width, height) { // 2.0 API
+  return new Canvas(width, height);
```

Comment:
this is important to interface out for when we want to bump to `canvas@2.0`, will make changing it easier, thanks!

Reply:
Yeah, saw you were doing that, so just applying to tests...